packages/core/src/components/RadioButton/RadioButtonRow.tsx

```diff
@@ -112,12 +116,10 @@ const RadioButtonRow: React.FC<RadioButtonRowProps & IconSlot> = ({
       >
         <RadioButton
           Icon={Icon}
-          selected={
-            selected || (contextValue != null && contextValue === value)
-          }
+          selected={isSelected ? true : false}
+          value={realValue}
           color={color}
           unselectedColor={unselectedColor}
-          onPress={handlePress}
```

Comment:
where is this onPress being called if its not here?

Reply:
@peterpme  It's at the topmost level of this component tree with the `Touchable` line 95 ... I had found some issues conflicting having both the Row control it and the Button control the component.